src/hooks/v2/V2ContractLoader.ts

```diff
@@ -73,12 +78,40 @@ const loadContract = async (
   network: NetworkName,
   signerOrProvider: JsonRpcSigner | JsonRpcProvider,
 ): Promise<Contract | undefined> => {
-  const contractOverride = CONTRACT_ABI_OVERRIDES[contractName]
-  const version = contractOverride?.version ?? 'latest'
-  const filename = contractOverride?.filename ?? contractName
-
-  const contract = await import(
-    `@jbx-protocol/contracts-v2-${version}/deployments/${network}/${filename}.json`
-  )
-  return new Contract(contract.address, contract.abi, signerOrProvider)
+  let contractJson: { abi: ContractInterface; address: string } | undefined =
```

Comment:
new pattern for importing contracts from multiple sources.

note: we aren't able to import ENS contracts from packages, so those contract artifacts have been added to src/constants for now